src/bindings/python/src/pyopenvino/pyopenvino.cpp

```diff
@@ -157,6 +160,38 @@ PYBIND11_MODULE(_pyopenvino, m) {
                 serialize(model, xml_path="./serialized.xml", bin_path="./serialized.bin", version="IR_V11")
         )");
 
+    m.def(
+        "save_model",
+        [](std::shared_ptr<ov::Model>& model,
+           const py::object& xml_path,
+           bool compress_to_fp16) {
+            ov::save_model(model,
+                          Common::utils::convert_path_to_string(xml_path),
+                          compress_to_fp16);
+        },
+        py::arg("model"),
+        py::arg("output_model"),
```

Comment:
Wouldn't it better to name this argument `xml_path` to be aligned with `serialize`?
Or `output_path` -- it's more self-explainable

CC @jiwaszki

Reply:
This new naming is explained by:
1. Align with a similar parameter in new `ovc` tool, this is also `output_model` that is complimentary to `input_model`.
2. Hiding details about IR implementation: `xml`. We can change xml to something else in the future when go away from xml format.

Reply:
We can adjust naming in a separate PR. Will collect more feedback about it. But even now it has "story" underneath as explained.